#Cargar y visualizar datos con Pandas (CSV en carpeta de Colab)

### Objetivos

* Cargar y visualizar archivos CSV en Google Colab usando Pandas.

* Practicar operaciones básicas de inspección y visualización de datos.

### ¿Qué vamos a hacer?

* Cargar ventas.csv (subido manualmente al entorno de Colab) y ver sus primeras filas.

> Tip: en Colab podés subir archivos con el ícono de carpeta (panel izquierdo) o con files.upload().

## Preparación del entorno

Antes de trabajar con archivos de datos, vamos a importar las librerías que usaremos en este cuaderno:

- **Pandas**: para cargar, manipular y visualizar datos en tablas (DataFrames).
- **Matplotlib**: para realizar gráficos sencillos.
- **Pathlib**: para manejar rutas de archivos de forma más clara.

Además, configuramos algunas opciones para que las tablas se vean completas en Colab.


In [10]:
import pandas as pd
from pathlib import Path

# Opciones de display para que se vea cómodo en Colab
pd.set_option("display.max_rows", 20)
pd.set_option("display.max_columns", 50)
pd.set_option("display.width", 120)

print("Pandas versión:", pd.__version__)

Pandas versión: 2.2.2


## Cargar un archivo CSV y ver sus primeras filas

Ahora vamos a cargar el archivo **ventas.csv**, que ya subimos manualmente a la carpeta del entorno de Google Colab (normalmente `/content`).

Con el método `pd.read_csv()` de Pandas podemos leer el archivo y guardarlo en un `DataFrame`, que es como una tabla en memoria.

Después, usamos:
- `.shape` para conocer la cantidad de filas y columnas.
- `.head()` para ver las primeras 5 filas de la tabla.


In [11]:
# Definimos la ruta del archivo
ruta_csv = Path("ventas.csv")

# Cargamos el archivo CSV en un DataFrame
df_ventas = pd.read_csv(ruta_csv)

# Mostramos información básica
print("Tamaño del DataFrame (filas, columnas):", df_ventas.shape)
df_ventas.head()


Tamaño del DataFrame (filas, columnas): (954, 5)


Fecha     Vendedor Sede Entrega    Producto     Importe
0  06/11/2020  Vendedor 10       Sede D  Producto Z   $75880.72
1  01/10/2021  Vendedor 10       Sede D  Producto Y  $279412.05
2  09/05/2021   Vendedor 4       Sede A  Producto W         NaN
3  07/10/2021   Vendedor 3       Sede D  Producto U   $33830.98
4  02/08/2021   Vendedor 3       Sede C  Producto U  $205706.84

**Observaciones:**
- El DataFrame tiene 954 filas y 5 columnas.
- La columna **Importe** contiene el símbolo `$`. Esto hace que Pandas la lea como texto.
- En algunos casos aparecen valores `NaN`, que indican datos faltantes.

Más adelante vamos a aprender cómo **limpiar y convertir** estas columnas.
Por ahora, lo importante es que ya sabemos cargar y visualizar un CSV.
